### PR TITLE
Update Python SDK & bash script

### PR DESCRIPTION
**Pull request by Sdddell**

### CODE REVIEW

File: extract_parse.sh [hunk 1]
```diff
@@ -11,16 +11,16 @@ apiKey="$1"
 func="$2"
 file_path="$3"
 
-upload
 if [ "$func" == "extract" ]; then
+    upload
```
Comment (CambioML) [inline extract_parse.sh:15]: qq: should upload out of the if, elif, so upload will also `apply` for parse logic?

**Comment by CambioML** [comment]
qq: do we run a quick test for all notebooks in this repo after the change?

File: extract_parse.sh (hunk 1 above)
Comment (CambioML) [reply extract_parse.sh:15]: nvm. It looks like this is a bug in previous code.

**Review by CambioML — APPROVED** [review]
Let's double check to make sure our notebook is still working as expected. For next PR, if you can add in the comment regarding what have been tested, it will help ease the review process a lot.

**Comment by Sdddell** [comment]
I have tested all the examples, they worked fine except `extract_key_values_from_cbc_reports_in_different_layout`.

**Review by CambioML — APPROVED** [review]

**Pull request merged**